lib/customer.rb

```diff
@@ -0,0 +1,44 @@
+
+require 'csv'
+
+# Class that holds customer information
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+  
+  def initialize(id, email_address, delivery_address)
+    @id = id
+    @email= email_address
+    @address = delivery_address
+  end
+  
+  # Load data in CSV to Customer class
+  def self.all
+    file = CSV.read('data/customers.csv').map(&:to_a)
+    
+    all_customers  = file.map do |new_c|
```

Comment:
I think you probably meant to return the result of this `map` instead of assigning it and relying on implicit return.

```suggestion
    return file.map do |new_c|
```

I spotted this because `rake` spit out this warning:
```
/Users/kaida/Developer/ada/cohort-13/grocery-store/lib/customer.rb:19: warning: assigned but unused variable - all_customers
```